communications/newsletters/newsletter_33_Apr2022.md

```diff
@@ -0,0 +1,259 @@
+
+Batool Almarzouq has also been working with Outreachy applicants, who will be working to translation chapters of _The Turing Way_ into Arabic.
+Outreachy provides paid remote, three-month internships to support diversity in Free and Open Source Software (FOSS).
+[Learn more about the Outreachy program here](https://www.outreachy.org/).
+Good luck to the applicants, we can't wait to see what you'll accomplish together!
+
+## Talks and Workshops
+
+### AI-UK
+
+![Alt:Malvika and Kirstie giving online presentation at AI-UK conference](https://i.imgur.com/PUQWMl8.jpg)
+_Discussing the Turing Way, and the benefit of community-led, open research practices at AI-UK._
+
+Kirstie Whitaker and Malvika Sharan, the co-leads of the project, spoke about the impact of _The Turing Way_ and reproducible research at [AI-UK](https://www.turing.ac.uk/ai-uk).
+AI-UK is the UK's national showcase of artificial intelligence (AI) and data science research and collaboration that took place on the 22 and 23 of March. While the event wasn't recorded, [you can check out their slides here](https://zenodo.org/record/6378011).
+
+### Collaborations Workshop
+
+Many of our community members were at [Collaborations Workshop](https://software.ac.uk/cw22) this year, which took place from 4 April - 7 April.
+Hosted by the [Software Sustainability Institute](https://software.ac.uk/), #CollabW22 brought together an interdisciplinary group of people to explore best practices and the future of research software.
+
+![Alt:Ester Plomp at Collaborations Workshop](https://i.imgur.com/3mMIeYX.jpg)
+_Thank you for supporting our community, Esther!_
+
+- Esther Plomp gave a lightning talk presentation called "The Turing Way: Reflection on the community-led development since 2019".
+You can view [the slide here](https://doi.org/10.6084/m9.figshare.19461026).
+
+![Alt:Panelists at Collaborations Workshop](https://i.imgur.com/Mb3sGDO.jpg)
+_The panel discussed the implementations of ethics in research and software, and give practical guidance to workshop participants on how ethical considerations can strengthen the work they are doing._
+
+- Arielle Bennett joined a Panel on Ethics, chaired by Laura Carter.
+Both are authors and mentors of several chapters in our [Guide for Ethical Research](https://the-turing-way.netlify.app/ethical-research/ethical-research.html?highlight=ethics).
+[Watch the recording here](https://www.youtube.com/watch?v=AfotGso8rDI)!
+
+![Alt:Screenshots of Slack posts and presentation titles](https://i.imgur.com/NYfMx7q.jpg)
+_Congratulations to the participants and presenters at #CW22!_
+
+- Achintya Rao and his team members developed a [Carpentries-style](https://carpentries.org/) lesson on R's bug tracking process, that was awarded third place during the hack day.
+- Malvika Sharan and her team members developed collaboration project idea, "Hi-Brid! Leaving no one behind in hybrid working!" that won second place during the Collaborative Idea's session.
```

Comment:
"developed the collaboration project idea"

Reply:
Thanks for catching that, Arielle!

Reply:
```suggestion
- Malvika Sharan and her team members developed a collaborative project idea, "Hi-Brid! Leaving no one behind in hybrid working!" that won second place during the Collaborative Idea's session.
```